packages/go_router/example/lib/books/main.dart

```diff
@@ -62,9 +62,10 @@ class Bookstore extends StatelessWidget {
         redirect: (_) => '/books/popular',
       ),
       GoRoute(
-        path: '/book/:bookId',
-        redirect: (GoRouterState state) =>
-            '/books/all/${state.params['bookId']}',
+        path: '/book/:kind/:bookId',
```

Comment:
are we no longer supporting `/book/:bookId`?

Reply:
That's my understanding, yes. The state needs to be in the URL in order to be preserved.

Reply:
no a hard requirement inorder to fix this, why not keep both redirects?